### ISSUE TITLE
Support for Windows high contrast mode

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#4297]
Follow up to https://github.com/WordPress/gutenberg/issues/1562#issuecomment-313476750 and complementing a similar ticket open on the core Trac, see https://core.trac.wordpress.org/ticket/41286

Also mentioned in the (now closed) issue with initial accessibility recommendations, see #297 point 5:
> You will have to support Windows High Contrast Mode in your controls, including the SVGs for the buttons. Luckily, this may not be very hard for single-color SVGs as the fill can be defined via a media query.

Windows high contrast mode removes, among other things, any background and box-shadow. Here's an example on Windows 10 and IE 11 with some text selected:

![screenshot 24](https://user-images.githubusercontent.com/1682452/34576487-54dbcdc6-f17e-11e7-9ce5-5606888ed6bb.png)

Overall, the interface is visible but the biggest issue is the focus style. All the ones based on box-shadow are not visible, including, for example, some important controls in the toolbar and all the sidebar panels.

For many other controls, the focus indication is actually given only by the text of the tooltip:

![screenshot 25](https://user-images.githubusercontent.com/1682452/34576579-a0d40388-f17e-11e7-854e-a812d54c4eeb.png)

As mentioned in the Trac ticket, and as a long term goal, it would be great to switch to a new focus style not exclusively built with box-shadow. Note: borders and outlines are visible in high contrast mode.
